Source/OxyPlot.Wpf/Utilities/PixelLayout.cs

```diff
@@ -0,0 +1,114 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="PixelLayout.cs" company="OxyPlot">
+//   Copyright (c) 2020 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Provides functionality to support non-default dpi scaling
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.Wpf.Utilities
```

Comment:
we have not used namespaces like this, but I agree this is a better way

Reply:
Hmm, I was sure I checked it... Should be changed. Is it possible to fix it via github interface (2 file changes (commits?) + squash -> the same pull request)? (sry, 10 yrs of TFS made me somehow resistant to understanding GIT workflow) Or it's simpler close this request and open a new?